src/Components/LandingPage/LandingPage.js

```diff
@@ -13,81 +20,110 @@ import {
-                </Text>
-                <Button
-                  component="a"
-                  target="_blank"
-                  variant="link"
-                  icon={<ExternalLinkAltIcon />}
-                  iconPosition="right"
-                  isInline
-                  href={
-                    'https://www.osbuild.org/guides/image-builder-service/architecture.html'
-                  }
-                >
-                  Repositories
-                </Button>
-              </TextContent>
+            <HelpIcon />
+          </Button>
+        </Popover>
+        <Popover
+          headerContent={'About open source'}
+          bodyContent={
+            <TextContent>
+              <Text>
+                This service is open source, so all of its code is inspectable.
+                Explore repositories to view and contribute to the source code.
+              </Text>
+              <Button
+                component="a"
+                target="_blank"
+                variant="link"
+                icon={<ExternalLinkAltIcon />}
+                iconPosition="right"
+                isInline
+                href={
+                  'https://www.osbuild.org/guides/image-builder-service/architecture.html'
+                }
+              >
+                Repositories
+              </Button>
+            </TextContent>
+          }
+        >
+          <Button
+            variant="plain"
+            aria-label="About Open Services"
+            className="pf-u-pl-sm header-button"
+          >
+            <CodeBranchIcon />
+          </Button>
+        </Popover>
+      </PageHeader>
+      <section className="pf-l-page__main-section pf-c-page__main-section">
+        {!isBeta() && showBetaAlert && (
+          <Alert
+            className="pf-u-mb-xl"
+            isInline
+            variant="default"
+            title="Build and launch AWS or Azure systems with custom content now available in our Beta environment."
+            actionClose={
+              <AlertActionCloseButton onClose={() => setShowBetaAlert(false)} />
+            }
+            actionLinks={
+              <Button
+                isInline
+                component="a"
+                variant="link"
+                href="/beta/insights/image-builder/landing"
+              >
+                <BetaLabel />
+                Go to Beta
+              </Button>
             }
           >
-            <Button
-              variant="plain"
-              aria-label="About Open Services"
-              className="pf-u-pl-sm header-button"
-            >
-              <CodeBranchIcon />
-            </Button>
-          </Popover>
-        </PageHeader>
-        <section className="pf-l-page__main-section pf-c-page__main-section">
-          <ImagesTable />
-        </section>
-        <Outlet />
-      </React.Fragment>
-    );
-  }
-}
+            <p>
+              Create customized operating system images, launch hosts based on
```

Comment:
Update to
Launch Amazon Web Services or Microsoft Azure hosts to the cloud from the console.<br>
Link custom repositories and build any supported image with custom content.